media/css/firefox/whatsnew/whatsnew-56.scss

```diff
@@ -0,0 +1,388 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+'../../pebbles/includes/lib',
+'../../hubs/sections';
+
+@mixin photon-horizontal-rule() {
+    padding-top: 40px;
+    position: relative;
+
+    &:before {
+        background: #00c8d7 linear-gradient(to left, #0080ff 0%, #00feff 0%, #06fde1 60%, #21fa64 75%, #33f70c 100%);
+        border-radius: 5px;
+        content: '';
+        height: 5px;
+        left: 50%;
+        margin-left: -65px;
+        position: absolute;
+        top: 0;
+        width: 130px;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+body {
+    @include open-sans;
+    overflow-x: hidden;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: normal;
+}
+
+#outer-wrapper {
+    overflow-x: hidden;
+}
+
+/* -------------------------------------------------------------------------- */
+// Up-to-date top header
+
+.up-to-date-header {
+    text-align: center;
+
+    h1 {
+        @include font-size-level4;
+        background: url('/media/img/firefox/whatsnew_56/green-check.svg') left center no-repeat;
+        color: #4a4a4a;
+        display: inline-block;
+        font-weight: bold;
+        margin: 0 auto;
+        padding: 12px 0 10px 60px;
+    }
+}
+
+html[dir="rtl"] .up-to-date-header {
+    h1 {
+        background-position: right center;
+        padding: 12px 60px 10px 0;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Content intro
+
+.intro {
+    background: #0060df;
+    background: url('/media/img/firefox/whatsnew_56/fox-tail-header.png') top -340px center no-repeat,
```

Comment:
Nit - do we need a fallback `background` declaration here? Should we specify `background-color` and `background-image`?

Reply:
Ah this was copy pasta from the `/quantum` page, where we did need a fallback. Removing here 👍